macros/base/probing/dockable_probe.cfg

```diff
@@ -211,6 +211,10 @@ gcode:
 
     {% set probe_servo_enabled = printer["gcode_macro _USER_VARIABLES"].probe_servo_enabled %}
 
+    # Probe retainer
+    {% set variable_probe_dock_separate_retainer = printer["gcode_macro _USER_VARIABLES"].probe_dock_separate_retainer %}

```

Comment:
Can you add a default of False here?

Reply:
Sure, resolved here: 01fded42209461f531a22cd8c4bfa2eba6236b5c